testing/ubuntu/presubmit/run_if_changed.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+### Ignore this test if there are no relevant changes
+cd ${CMLE_REPO_DIR}/${CMLE_TEST_BASE_DIR}
+
+DIFF=`git diff master $KOKORO_GITHUB_PULL_REQUEST_COMMIT $PWD`
```

Comment:
No. Note the `$PWD` on the end of the diff statement. That assures that this will only be processed if there are relevant changes in the `CMLE_TEST_BASE_DIR` directory. In this case `flowers/`

Assuming every presubmit follows this strategy, only tests for changed directories will be run.